tests/test_sc_sets/test_sc_numbered_set.py

```diff
@@ -0,0 +1,135 @@
+"""
+This source file is part of an OSTIS project. For the latest info, see https://github.com/ostis-ai
+Distributed under the MIT License
+(See an accompanying file LICENSE or a copy at https://opensource.org/licenses/MIT)
+"""
+
+from sc_client.client import check_elements, template_search
+from sc_client.constants import sc_types
+from sc_client.models import ScAddr, ScTemplate
+
+from sc_kpm import ScKeynodes
+from sc_kpm.sc_sets.sc_numbered_set import ScNumberedSet
+from sc_kpm.utils.common_utils import create_link, create_node, create_nodes
+from tests.common_tests import BaseTestCase
+
+
+class ScNumberedSetTestCase(BaseTestCase):
+    def test_create_with_set_node(self):
```

Comment:
How are you test code coverage?

Reply:
I did it locally

Reply:
![image](https://user-images.githubusercontent.com/90003166/235632380-7eb3ad56-2b63-42a2-be1b-da5978772fbf.png)

Reply:
Tests will be added in other PR